src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenStore.cs

```diff
@@ -80,6 +84,8 @@ public void SaveCookieToken(HttpContext httpContext, string token)
             }
             SetCookiePath(httpContext, options);
 
+            _options.ConfigureCookieOptions?.Invoke(options);
```

Comment:
Yea, not changing the existing behaviour. Calling the configure delegate here allows the user to overwrite what we set but as @rynowak mentioned, they will be doing so at their own risk.